proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,157 +44,127 @@ direction of the policies set by the Board.
-
- * A nomination requires a maximum one (1) page nomination pitch which should include the nominees name, contact information and supporting statement identifying the nominees experience in CNCF domains.
-
- * The Governing Board, End User TAB and TOC shall determine the timeline and dates for nominations, voting, and any other details regarding the nomination and election process for their TOC elections.
-
- * A minimum of 14 calendar days shall be reserved for an Evaluation Period whereby TOC nominees may be contacted and/or evaluate the electors.
-
- * Elections: After the Evaluation Period, the Governing Board and the seven non-TOC-appointed TOC members, shall vote on each nominee individually. A valid vote shall require as many votes as would be required to meet quorum for a meeting. Each nominee shall require support of over 50% majority of those voting to validate the nominee meets the qualification criteria. Nominees passing with a majority vote shall be Qualified Nominees.
-
-  * If the number of Qualified Nominees are equal to or less than the number of TOC seats available to be elected, the Qualified Nominees shall be approved after the nomination period has closed. If there are more Qualified Nominees than open TOC seats available for election by either the the Governing Board, End User TAB or TOC, then that group shall elect the TOC members via a Condorcet vote. A Condorcet vote shall be run using the Condorcet-IRV method through the Cornell online service ([*http://civs.cs.cornell.edu/*](http://civs.cs.cornell.edu/)).
-
-  * If there are fewer Qualified Nominees than open TOC seats available for election by either the Governing Board, End User TAB or TOC, the group shall initiate another round of nominations with each member or individual eligible to nominate submitting at most one (1) nominee.
-
-* Constraints
-
-  *  TOC Members shall serve two-year, staggered terms. The initial six elected TOC members from the Governing Board election shall serve an initial term of three (3) years. The TOC members initially elected by the End User TAB and TOC shall serve an initial term of two (2) years.
-
- * TOC members may be removed by a two-thirds vote of the other TOC members, with the impacted individual ineligible to participate in the vote.
-
- * Any TOC member that misses three (3) consecutive meetings shall be automatically suspended from eligibility to vote until having attended two meetings consecutively. For avoidance of doubt, the suspended TOC member shall be eligible to vote in the second consecutive meeting.
-
- * The TOC charter, model, approach, composition, etc may be modified by a two-thirds vote of the entire Governing Board.
-
-===========
-
+The CPC is an inclusive group with two types of members:
+
+* Regular members
+* Voting members
+
+### Regular members
```

Comment:
I still think that we don't need this membership class. Much less friction to just make the meetings open and have explicit voting members

Reply:
I think the meetings can still be open. I think there is a difference between people who simply want to observe/attend versus those who are going to participate actively.  It is much easier to word things as a responsibility of the CPC when there are Regular members.  It's harder to say X is a responsibility of whoever just happens to show up at meetings.  At the same time we want it to be inclusive so we don't want to say X is a responsibility of the CPC where that implies just the subset that would be the voting members in what is outlined here.  

I'd be happy to add wording that makes it clear that meetings will be open and that anybody can participate, but I think having the concept of regular members does add value.  We could possibly introduce "observers" who are free to attend but who don't necessarily have any responsibilities.

Reply:
I've updated with https://github.com/nodejs/bootstrap/pull/65/commits/5a6138e48b7177f2252d5519a786421ce02def45.

To test out how it would be without regular members, just drop out "Voting Members" from the types of participants and references to "Voting members".  I think it would the read like the smaller CPC (what was just the Voting members") is responsible for all of that as opposed the larger group of people we'd like to be involved.  I really do think there should be a difference between people who just want to observer versus actively participate in an ongoing manner.

Reply:
We should discuss the balance between autonomy/enforcement in the next meeting as well as the interpersonal conflict side of things.  I think that is needed before I'd be comfortable in adding too much on that front.